.github/ISSUE_TEMPLATE/new_node_package.yml

```
@@ -46,6 +46,13 @@ body:
       placeholder: ex. 0.0.0.20240513
     validations:
       required: true
+  - type: input
+    id: arguments
+    attributes:
+      label: Arguments
+      description: |	
+        Command-line arguments to pass to the tool when running it from the `Tools` directory. For example `--help`.
+      placeholder: ex. --help
```

> @Ana06 instead of default I added placeholder to be consistent with the other fields, also for some reason I didn't get it to display it

**We use placeholder to display an example without filling the field. What I was suggesting here is to have it pre-filled** as in many cases `--help` is the right value, so that it is pre-filled. `default` is the wrong attribute, as it is available for `dropdown`, but not for `input` type. **You should use `value`**. See GH documentation for more details: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-for-input.
```suggestion
      placeholder: ex. --help
```

This applies to the other templates too.